team.md

```diff
@@ -27,7 +27,7 @@ of Cloud Haskell as a whole. [Edsko De Vries][13], a member of Well-Typed and th
 author of much of the new implementation we have today, is still closely involved
 as well.
 
-[Tim][6] is the primary author and maintainer of [disributed-process-platform][8];
+[Tim][6] is the primary author and maintainer of [disributed-process][8];
```

Comment:
I think if we will add a paragraph about platform and core - it will be easy to write. Something like:

Most of the packages are maintained my the haskell-distributed organization(?) that with people from Well Typed, Tweag, and (other companies that are involved) and individuals (names). 
Main maintaiters are @facundominguez maintains core packages, @hyperthunk maintains platform packages.

Some libraries are maintained my third parties, most notably d-p-p2p.